python/amico/models.py

```diff
@@ -398,28 +397,41 @@ def resample( self, in_path, idx_out, Ylm_out ) :
 
 
     def fit( self, y, dirs, KERNELS, params ) :
+        singleb0 = True if len(y) == (1+self.scheme.dwi_count) else False
```

Comment:
I think that, for the moment, it is better to handle this b0-merging the same way it is done in COMMIT. This would improve consistency and code management from now on. Then, sure, one day we need to optimize this part in both AMICO and COMMIT. So, I would go for changing the data and kernels once we load them